src/store/flow/block/set-contact-property.ts

```diff
@@ -0,0 +1,76 @@
+import {IRootState} from '@/store'
+import {findBlockOnActiveFlowWith as findBlock, IBlock, IContext, SetContactProperty} from '@floip/flow-runner'
+import {ActionTree, MutationTree} from 'vuex'
+import {IFlowsState} from '../index'
+
+export interface ISetContactPropertyIndex {
+  blockId: IBlock['uuid'],
+  key: SetContactProperty['property_key'],
+}
+
+export interface ISetContactPropertyIndexWithValue extends ISetContactPropertyIndex {
+  value: SetContactProperty['property_value'],
+}
```

Comment:
Nice work with types and the whole job with the new add/remove/set/change interface 👍